cmd/lncli/commands_test.go

```diff
@@ -68,3 +73,64 @@ func TestParseChanPoint(t *testing.T) {
 		}
 	}
 }
+
+// TestParseTimeLockDelta tests parseTimeLockDelta with various
+// valid and invalid input values and verifies the output.
+func TestParseTimeLockDelta(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		timeLockDeltaStr      string
+		expectedTimeLockDelta uint16
+		expectErr             bool
+		expectedErrContent    string
+	}{
+		{
+			"-1",
```

Comment:
please use named variables. You then also dont need to set default values :)

Reply:
Thank you. :heart: